src/hb-ot-name-table.hh

```diff
@@ -108,6 +108,17 @@ struct NameRecord
     return_trace (out);
   }
 
+  bool isUnicode () const
+  {
+    unsigned int p = platformID;
+    unsigned int e = encodingID;
+    
+    if (p == 0 ||
+        (p == 3 && (e == 0 || e == 1 || e == 10))) return true;
```

Comment:
can simplify this to:

return (p == 0 ||
        (p == 3 && (e == 0 || e == 1 || e == 10)));